packages/gestalt-datepicker/src/InternalDateField.js

```diff
@@ -0,0 +1,285 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable react/display-name */
+// @flow strict-local
+import { useState, forwardRef, type Node } from 'react';
+import { DatePicker as MUIDatePicker } from '@mui/x-date-pickers';
+import { unstable_useDateField as useDateField } from '@mui/x-date-pickers/DateField';
```

Comment:
why is this unstable? Is that how they have it defined

Reply:
From MUI docs